Binary Search Trees/Binary Search Tree Implementation Complete.py

```diff
@@ -0,0 +1,202 @@
+class Node:
+    def __init__(self,value):
+        self.value = value
+        self.left = None
+        self.right = None
+    
+    def insert(self, value):
+        if self.value == value:
+            return
+        if self.value > value:
+            if self.left:
+                return self.left.insert(value)
+            else:
+                self.left = Node(value)
+        if self.value < value:
```

Comment:
This could be elif, currently it checks for `self.value < value` even if `self.value > value` was true in the previous check.